shell/package.json

```diff
@@ -41,51 +41,51 @@
     "@types/is-url": "1.2.30",
     "@types/node": "20.10.8",
     "@types/semver": "^7.5.8",
-    "@typescript-eslint/eslint-plugin": "~5.4.0",
-    "@typescript-eslint/parser": "~5.4.0",
+    "@typescript-eslint/eslint-plugin": "5.62.0",
+    "@typescript-eslint/parser": "5.62.0",
     "@vue/cli-plugin-babel": "~5.0.0",
     "@vue/cli-plugin-typescript": "~5.0.0",
-    "@vue/cli-service": "~5.0.0",
+    "@vue/cli-service": "5.0.8",
     "@vue/test-utils": "~2.0.0-0",
-    "@vue/vue3-jest": "^27.0.0-alpha.1",
+    "@vue/vue3-jest": "27.0.0",
     "add": "2.0.6",
     "ansi_up": "5.0.0",
-    "axios": "0.21.4",
     "axios-retry": "3.1.9",
+    "axios": "0.21.4",
     "babel-eslint": "10.1.0",
     "babel-plugin-module-resolver": "4.0.0",
     "babel-preset-vue": "2.0.2",
     "cache-loader": "4.1.0",
     "clipboard-polyfill": "4.0.1",
-    "color": "4.2.3",
-    "codemirror": ">=5.64.0 <6",
     "codemirror-editor-vue3": "2.7.1",
-    "cookie": "0.7.0",
+    "codemirror": ">=5.64.0 <6",
+    "color": "4.2.3",
     "cookie-universal": "2.2.2",
-    "core-js": "3.25.3",
+    "cookie": "0.7.0",
+    "core-js": "3.40.0",
     "cron-validator": "1.3.1",
     "cronstrue": "2.50.0",
     "cross-env": "6.0.3",
     "css-loader": "6.7.3",
     "csv-loader": "3.0.3",
     "custom-event-polyfill": "^1.0.7",
-    "cypress": "10.3.1",
-    "d3": "7.3.0",
+    "cypress": "9.7.0",
```

Comment:
Sorry for showing the question like this but, do we need Cypress in shell?

Reply:
I cannot answer that and I don't intend to fix it in this PR. Perhaps @aalves08 can provide more details.